### PR TITLE
HOMS-140 Save files from API

### PR DESCRIPTION
**Pull request by GideonEXE**

### CODE REVIEW

File: app/models/order.rb [hunk 1]
```diff
@@ -10,6 +10,7 @@ def id_from_code(code)
 
   belongs_to :order_type
   belongs_to :user
+  has_many :attachment
```
Comment (sugarfree1) [inline app/models/order.rb]: attachments?

File: config/initializers/minio.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+require 'aws-sdk'
+
+Rails.application.config.to_prepare do
+  config = Rails.application.config.app.fetch(:minio, {})
+  if !config.empty?
```
Comment (sugarfree1) [inline config/initializers/minio.rb]: unless?

File: lib/minio/adapter.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  class Adapter
+    extend Imprint::Mixin
+
+    def save_file(file, order_id)
+      s3 = Aws::S3::Resource.new(Aws::S3::Client.new)
+      bucket = s3.bucket(Rails.application.config.app[:minio][:bucket])
+      obj = bucket.object(file[:name])
+      obj.put(body: Base64.decode64(file[:content]))
+
+      attachment = Attachment.new({url: obj.public_url,
```
Comment (sugarfree1) [inline lib/minio/adapter.rb]: No need to put curly brackets.

**Comment by GideonEXE** [comment]
UPD

**Pull request merged**